host/allwinner/a20.go

```diff
@@ -0,0 +1,219 @@
+	"PD3":  {"LCD0_D3", "LVDS0_VN1"},
+	"PD4":  {"LCD0_D4", "LVDS0_VP2"},
+	"PD5":  {"LCD0_D5", "LVDS0_VN2"},
+	"PD6":  {"LCD0_D6", "LVDS0_VPC"},
+	"PD7":  {"LCD0_D7", "LVDS0_VNC"},
+	"PD8":  {"LCD0_D8", "LVDS0_VP3"},
+	"PD9":  {"LCD0_D9", "LVDS0_VN3"},
+	"PD10": {"LCD0_D10", "LVDS1_VP0"},
+	"PD11": {"LCD0_D11", "LVDS1_VN0"},
+	"PD12": {"LCD0_D12", "LVDS1_VP1"},
+	"PD13": {"LCD0_D13", "LVDS1_VN1"},
+	"PD14": {"LCD0_D14", "LVDS1_VP2"},
+	"PD15": {"LCD0_D15", "LVDS1_VN2"},
+	"PD16": {"LCD0_D16", "LVDS1_VPC"},
+	"PD17": {"LCD0_D17", "LVDS1_VNC"},
+	"PD18": {"LCD0_D18", "LVDS1_VP3"},
+	"PD19": {"LCD0_D19", "LVDS1_VN3"},
+	"PD20": {"LCD0_D20", "CSI1_MCLK"},
+	"PD21": {"LCD0_D21", "SMC_VPPEN"},
+	"PD22": {"LCD0_D22", "SMC_VPPPP"},
+	"PD23": {"LCD0_D23", "SMC_DET"},
+	"PD24": {"LCD0_CLK", "SMC_VCCEN"},
+	"PD25": {"LCD0_DE", "SMC_RST"},
+	"PD26": {"LCD0_HSYNC", "SMC_SLK"},
+	"PD27": {"LCD0_VSYNC", "SMC_SDA"},
+	"PE0":  {"TS0_CLK", "CSI0_PCLK"},
+	"PE1":  {"TS0_ERR", "CSI0_MCLK"},
+	"PE2":  {"TS0_SYNC", "CSI0_HSYNC"},
+	"PE3":  {"TS0_DLVD", "CSI0_VSYNC"},
+	"PE4":  {"TS0_D0", "CSI0_D0"},
+	"PE5":  {"TS0_D1", "CSI0_D1"},
+	"PE6":  {"TS0_D2", "CSI0_D2"},
+	"PE7":  {"TS0_D3", "CSI0_D3"},
+	"PE8":  {"TS0_D4", "CSI0_D4"},
+	"PE9":  {"TS0_D5", "CSI0_D5"},
+	"PE10": {"TS0_D6", "CSI0_D6"},
+	"PE11": {"TS0_D7", "CSI0_D7"},
+	"PF0":  {"SDC0_D1", "", "JTAG_MS1"},
+	"PF1":  {"SDC0_D0", "", "JTAG_DI1"},
+	"PF2":  {"SDC0_CLK", "", "UART0_TX"},
+	"PF3":  {"SDC0_CMD", "", "JTAG_DO1"},
+	"PF4":  {"SDC0_D3", "", "UART0_RX"},
+	"PF5":  {"SDC0_D2", "", "JTAG_CK1"},
+	"PG0":  {"TS1_CLK", "CSI1_PCLK", "SDC1_CMD"},
+	"PG1":  {"TS1_ERR", "CSI1_MCLK", "SDC1_CLK"},
+	"PG2":  {"TS1_SYNC", "CSI1_HSYNC", "SDC1_D0"},
+	"PG3":  {"TS1_DVLD", "CSI1_VSYNC", "SDC1_D1"},
+	"PG4":  {"TS1_D0", "CSI1_D0", "SDC1_D2", "CSI0_D8"},
+	"PG5":  {"TS1_D1", "CSI1_D1", "SDC1_D3", "CSI0_D9"},
+	"PG6":  {"TS1_D2", "CSI1_D2", "UART3_TX", "CSI0_D10"},
+	"PG7":  {"TS1_D3", "CSI1_D3", "UART3_RX", "CSI0_D11"},
+	"PG8":  {"TS1_D4", "CSI1_D4", "UART3_RTS", "CSI0_D12"},
+	"PG9":  {"TS1_D5", "CSI1_D4", "UART3_CTS", "CSI0_D13"},
+	"PG10": {"TS1_D6", "CSI1_D6", "UART4_TX", "CSI0_D14"},
+	"PG11": {"TS1_D7", "CSI1_D7", "UART4_RX", "CSI0_D15"},
+	"PH0":  {"LCD1_D0", "", "UART3_TX", "", "PH_EINT0"},
+	"PH1":  {"LCD1_D1", "", "UART3_RX", "", "PH_EINT1"},
+	"PH2":  {"LCD1_D2", "", "UART3_RTS", "", "PH_EINT2"},
+	"PH3":  {"LCD1_D3", "", "UART3_CTS", "", "PH_EINT3"},
+	"PH4":  {"LCD1_D4", "", "UART4_TX", "", "PH_EINT4"},
+	"PH5":  {"LCD1_D5", "", "UART4_RX", "", "PH_EINT5"},
+	"PH6":  {"LCD1_D6", "", "UART5_TX", "MS_BS", "PH_EINT6"},
+	"PH7":  {"LCD1_D7", "", "UART5_RX", "MS_CLK", "PH_EINT7"},
+	"PH8":  {"LCD1_D8", "ERXD3", "KP_IN0", "MS_D0", "PH_EINT8"},
+	"PH9":  {"LCD1_D9", "ERXD2", "KP_IN1", "MS_D1", "PH_EINT9"},
+	"PH10": {"LCD1_D10", "ERXD1", "KP_IN2", "MS_D2", "PH_EINT10"},
+	"PH11": {"LCD1_D11", "ERXD0", "KP_IN3", "MS_D3", "PH_EINT11"},
+	"PH12": {"LCD1_D12", "", "PS2_SCK1", "", "PH_EINT12"},
```

Comment:
Holly crap, that's really PS/2. That's amazing.

Reply:
:D